crates/store/src/errors.rs

```diff
@@ -40,65 +40,59 @@ pub enum NullifierTreeError {
 pub enum DatabaseError {
     // ERRORS WITH AUTOMATIC CONVERSIONS FROM NESTED ERROR TYPES
     // ---------------------------------------------------------------------------------------------
-    #[error("Account error: {0}")]
+    #[error("account error")]
     AccountError(#[from] AccountError),
-    #[error("Account delta error: {0}")]
+    #[error("account delta error")]
     AccountDeltaError(#[from] AccountDeltaError),
-    #[error("Block error: {0}")]
+    #[error("block error")]
     BlockError(#[from] BlockError),
-    #[error("Closed channel: {0}")]
+    #[error("closed channel")]
     ClosedChannel(#[from] RecvError),
-    #[error("Deserialization of BLOB data from database failed: {0}")]
-    DeserializationError(DeserializationError),
-    #[error("Hex parsing error: {0}")]
+    #[error("deserialization failed")]
+    DeserializationError(#[from] DeserializationError),
+    #[error("hex parsing error")]
     FromHexError(#[from] hex::FromHexError),
-    #[error("SQLite error: {0}")]
+    #[error("SQLite deserialization error")]
     FromSqlError(#[from] FromSqlError),
-    #[error("I/O error: {0}")]
+    #[error("I/O error")]
     IoError(#[from] io::Error),
-    #[error("Migration error: {0}")]
+    #[error("migration failed")]
     MigrationError(#[from] rusqlite_migration::Error),
-    #[error("Missing database connection: {0}")]
+    #[error("missing database connection")]
     MissingDbConnection(#[from] PoolError),
-    #[error("Note error: {0}")]
+    #[error("note error")]
     NoteError(#[from] NoteError),
-    #[error("SQLite error: {0}")]
+    #[error("SQLite error")]
     SqliteError(#[from] rusqlite::Error),
 
     // OTHER ERRORS
     // ---------------------------------------------------------------------------------------------
-    #[error("Account hashes mismatch (expected {expected}, but calculated is {calculated})")]
+    #[error("account hash mismatch (expected {expected}, but calculated is {calculated})")]
     AccountHashesMismatch {
         expected: RpoDigest,
         calculated: RpoDigest,
     },
-    #[error("Account {0} not found in the database")]
+    #[error("account {0} not found")]
     AccountNotFoundInDb(AccountId),
-    #[error("Accounts {0:?} not found in the database")]
+    #[error("accounts {0:?} not found")]
     AccountsNotFoundInDb(Vec<AccountId>),
-    #[error("Account {0} is not on the chain")]
+    #[error("account {0} is not on the chain")]
     AccountNotOnChain(AccountId),
```

Comment:
This should likely be renamed to `AccountNotPublic`

Reply:
Thank you that makes much more sense :)

Reply:
That said it doesn't appear to be used anywhere?